task-list/app/views/people/index.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<ul>
+<% @all_people.each do  |person| %>
+
+ <li><%= link_to person.name, person_path(person.id) %></li>
+    <p> uncompleted tasks: <%= person.tasks.where(completed_at: nil).count %> </p> <%#count is a rails method, not a ruby method! %>
+
```

Comment:
Nice job using the link_to rails helper, combined with named routes + passing in parameters!

In general, it's best to keep logic out of the views as much as possible, so I'd probably shift the "person.tasks.where(completed_at: nil).count" logic. It might make sense as a method within the Task model? A count_completed_tasks method that returns the number of completed tasks for a specific person instance? Then you can call that model method in the controller and pass the result as an instance variable, something like @num_completed_tasks